src/frontend/mame/clifront.cpp

```diff
@@ -568,27 +568,27 @@ void cli_frontend::listroms(const char *gamename)
 			osd_printf_info("\n");
 		first = false;
 		osd_printf_info("ROMs required for driver \"%s\".\n"
-				"Name                    Size Checksum\n", drivlist.driver().name);
+				"Name                                   Size Checksum\n", drivlist.driver().name);
```

Comment:
Please change this to be like:

```
    osd_printf_info("ROMs required for driver \"%s\".\n"
                    "%-32s %10s %s\n",
                    drivlist.driver().name, "Name", "Size", "Checksum");
```

Once again, being able to just update matching numbers is better than having to count spaces